shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -296,6 +1096,86 @@ - (void)testCompositePlatformView {
   XCTAssertTrue(CGRectEqualToRect(platformViewRectInFlutterView, CGRectMake(100, 100, 300, 300)));
 }
 
+- (void)testBackdropFilterCorrectlyPushedAndReset {
```

Comment:
This is the test for This if the fix for https://github.com/flutter/flutter/issues/109783